src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,28 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
 
+    private static final StorageDaoImpl STORAGE_DAO = new StorageDaoImpl();
```

Comment:
storageDao should not be named in UpperCase

Reply:
Why not? it is a [constant](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) field

Reply:
That's a better practice for interfaces
Constants look better with Strings, ints, etc...
Also, you should declare an interface, not implementation, and then initialize it with its implementation

Reply:
I agree thats better to use interface variable, but im not sure about naming better practice. Which sources are you use?